easybuild/easyconfigs/g/GitPython/GitPython-1.0.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'GitPython'
+version = '1.0.2'
+
+homepage = 'https://gitpython.readthedocs.org/en/stable/'
+description = """GitPython is a python library used to interact with git repositories, high-level like git-porcelain, or low-level like git-plumbing."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ('python -c"import %(ext_name)s"', '')
+
+# allow use of system Python
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+exts_list = [
+    ('setuptools', '18.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools/'],
+    }),
+    ('pip', '7.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pip/'],
+    }),
+    ('argparse', '1.3.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/argparse/'],
+    }),
+    ('smmap', '0.9.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/smmap/'],
+    }),
+    ('gitdb', '0.6.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gitdb/'],
+    }),
+    ('GitPython', '1.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/G/GitPython'],
+        'modulename': 'git',
+    }),
+    # Add the requirements for GitHub integration for EasyBuild
+    ('keyrings.alt', '1.1.1', {
```

Comment:
this package provides additional (potentially less secure) backends, but I don't think that should hold us back from including it